src/PersistentFilters.ts

```diff
@@ -230,4 +278,45 @@ export class PersistentFilters {
             await vscode.workspace.getConfiguration().update(this.schema, settings, vscode.ConfigurationTarget.Global);
         }
     }
+
+    /**
+     * Updates the recently-opened file history array in settings.json
+     */
```

Comment:
I think the block comment above + function name makes it clear and this comment is redundant.